### DISCUSSION TITLE
Kibana/Elastic error 2.3.xx

### DISCUSSION THREAD

**Discussion by Gamerjake8** [Security-Onion-Solutions/securityonion#4111] [Unsupported Versions] [answered]
We've tried building out 2.3.2 through 2.3.50 and we're experiencing the same error on every version.

In Elastic/Kibana we get 0 count for logs and these errors:
Could not locate that index-pattern-field (id: @timestamp)
Could not locate that index-pattern-field (id: event.category.keyword)
Could not locate that index-pattern-field (id: event.dataset.keyword)
Could not locate that index-pattern-field (id: event.module.keyword)
Could not locate that index-pattern-field (id: observer.name.keyword)

We are building out a Manager and Sensor in distributed environment on ESXI in a closed network. We're using a fresh install. We've verified that the sensor is capturing data. We've verified that the Manager is receiving data from the sensor. We've read through just about every forum. When we go in Stack Management > Kibana-Index Patterns > *:so-* (Default) we are only seeing these fields:

_id string Url
_index string
_score number
_source _source
_type string

Any help would be greatly appreciated.

**Comment by gcsrellis** [comment, marked as answer]
Do you have a Search node also?  Like Managersearch or a dedicated search node?

**Reply by Gamerjake8** [reply]
Negative. Just a Manager and a Sensor in a distributed environment.

**Reply by gcsrellis** [reply]
You will need to reconfigure your deployment to include a search node.  Either redeploy the manager as a managersearch or deploy a separate search node vm.
